.buildkite/pipeline_perf.py

```diff
@@ -90,7 +90,7 @@ def build_group(test):
 if not args.instances:
     args.instances = DEFAULT_INSTANCES
 if not args.kernels:
-    args.kernels = DEFAULT_KERNELS
+    args.kernels = DEFAULT_PLATFORMS
```

Comment:
we may need to also change how we pass this parameter then... maybe we need to use `nargs=2` in the parameter definition?

Reply:
Should be fixed now.